### PR TITLE
Helm Chart for 2.6.104

### PR DESCRIPTION
**Pull request by karok2m**

### CODE REVIEW

File: deploy/helm/azure-industrial-iot/Chart.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ home: https://github.com/Azure/Industrial-IoT
 maintainers:
 - name: Karapet Kostandyan
   email: kakostan@microsoft.com
-appVersion: 2.5.2
+appVersion: 2.6.62
```
Comment (cristipogacean) [inline deploy/helm/azure-industrial-iot/Chart.yaml]: shouldn't it be 2.6.10x or so?

File: deploy/helm/azure-industrial-iot/README.md [hunk 2]
```diff
@@ -490,6 +837,191 @@ azure:
...
 ```
+
+## Special Notes
+
+### Resource Requests And Limits
+
+Helm chart does not set any [resource requests or limits](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/)
+for Pod Containers. We recommend to set memory requests to at least `"64Mi"` and cpu requests to at least
+`"20m"`. If you set resource limits, be informed that some components might require up to `"256Mi"` of
+memory, such as `frontend`, `registry`, `publisher` and a few others.
```
Comment (cristipogacean) [inline deploy/helm/azure-industrial-iot/README.md]: >frontend [](start = 17, length = 8)

I think we use the term "engineering tool" for "frontend" in other places in documentation, I have no clue if this is right or not, but we should be consequent through documentation with naming of our components.

File: deploy/helm/azure-industrial-iot/values.yaml [hunk 3]
```diff
@@ -5,12 +5,12 @@ image:
   # image.registry is URL of Docker registry from where images will be pulled.
   registry: mcr.microsoft.com
   # image.tag defines which version of Docker images to pull.
-  tag: 2.5.2
+  tag: 2.6.104
```
Comment (cristipogacean) [inline deploy/helm/azure-industrial-iot/values.yaml]: >104 [](start = 11, length = 3)

can't we use :latest respective :preview here?

File: deploy/helm/azure-industrial-iot/Chart.yaml (hunk 1 above)
Comment (karok2m) [reply deploy/helm/azure-industrial-iot/Chart.yaml]: Fixed. It should be `2.6.104`.

File: deploy/helm/azure-industrial-iot/values.yaml (hunk 3 above)
Comment (karok2m) [reply deploy/helm/azure-industrial-iot/values.yaml]: We shouldn't. Helm best practices explicitly state not to use 'floating' image tags and to always prefer fixed ones.

We should keep in mind that in Kubernetes environment different components may fail and be restarted independently. Each restart can result in a new image being pulled and used. So having a floating tag may then result in a situation where you have components of several different versions in the same environment. Also, having a floating tag would mean not having reproducible deployments, which should also be avoided.

Comment (karok2m) [reply deploy/helm/azure-industrial-iot/values.yaml]: BTW, this does not mean that users cannot change the tag. Quite the opposite, it is very easy to override this value. But then users would have to explicitly do that at their own risk.

File: deploy/helm/azure-industrial-iot/README.md (hunk 2 above)
Comment (karok2m) [reply deploy/helm/azure-industrial-iot/README.md]: Good point, thanks. I will change it to engineering tool and `engTool` for configuration parameters. But I will keep `/frontend` as URL path base for engineering tool. Sound good ?

Comment (karok2m) [reply deploy/helm/azure-industrial-iot/README.md]: Decided to use `engineeringTool` for configuration as well.

**Pull request merged**